src/snapshotter.test.js

```diff
@@ -180,4 +180,30 @@ describe('The snapshotter', () => {
     await mockSnapshot.takeSnap();
     expect(logger.error.mock.calls.length).toBe(1);
   });
+
+  it('Returns the correct browser capabilities if mobile emulator is used', async () => {
+    const mockSnapshot = (SnapShotter.prototype.getMobileBrowserCapability = jest.fn());
+    new SnapShotter(
+      {
+        gridUrl: 'https://lol.com',
+        mobileDeviceName: 'test'
+      },
+      { webdriver, By, until }
+    );
+
+    expect(mockSnapshot.mock.calls.length).toBe(1);
+  });
+
+  it('Returns the correct browser capabilities if normal browser is used', async () => {
```

Comment:
don't mean to be overly fussy but 'correct browser capabilities' / 'normal browser' aren't the best descriptive words for the test :)

something more like below:
it should return desktop capabilities when called with a non-mobile browser

Reply:
cool will change :)